libqtile/backend/wayland/window.py

```diff
@@ -77,6 +79,7 @@ def __init__(self, core: Core, qtile: Qtile, surface: SurfaceType):
         self.surface = surface
         self._group: _Group | None = None
         self.popups: list[XdgPopupWindow] = []
+        self._idle_inhibitors: list[IdleInhibitorV1] = []
```

Comment:
If we don't need the objects themselves, maybe we can get away with just using an `int` as a counter, and increment/decrement it?

Reply:
Yep that is probably easier, commited.